src/shared/Microsoft.AzureRepos/AzureReposHostProvider.cs

```diff
@@ -278,6 +344,41 @@ private static string GetAccountNameForCredentialQuery(InputArguments input)
             throw new InvalidOperationException("Host is not Azure DevOps.");
         }
 
+        /// <summary>
+        /// Check if Azure DevOps Personal Access Tokens should be used or not.
+        /// </summary>
+        /// <returns>True if Personal Access Tokens should be used, false otherwise.</returns>
+        private bool UsePersonalAccessTokens()
+        {
+            // Default to using PATs whilst the Azure AT functionality is being tested
+            const bool defaultValue = true;
+
+            if (_context.Settings.TryGetSetting(
+                AzureDevOpsConstants.EnvironmentVariables.CredentialType,
+                KnownGitCfg.Credential.SectionName,
+                AzureDevOpsConstants.GitConfiguration.Credential.CredentialType,
+                out string valueStr))
```

Comment:
So, this loads from environment the same way it loads from the config? Then the docs are definitely wrong and should use `azure` as their value.